Source/Scene/ModelExperimental/InstancingPipelineStage.js

```diff
@@ -119,6 +125,34 @@ InstancingPipelineStage.process = function (renderResources, node, frameState) {
     instancingVertexAttributes
   );
 
+  if (instances.transformInWorldSpace) {
```

Comment:
I checked the coverage results, this branch was not taken. 

Could you add a unit test that the shader is updated correctly?